### ISSUE TITLE
Program crashes when doing two searches in a row

### ISSUE THREAD

**Issue by lamons** [Alex-Rose/fb-messenger-cli#47]
When I search by a name with upper case like `/search Mike` it went well but when I type a lower case word like `/search mike` the program crashes and return this error message:

```
/usr/local/lib/node_modules/fb-messenger-cli/scripts/search.js:23
    if(search.filtered[i].position == choice.toLowerCase().trim()){
                                            ^

TypeError: Cannot read property 'toLowerCase' of undefined
    at Search.selectConvo (/usr/local/lib/node_modules/fb-messenger-cli/scripts/search.js:23:45)
    at EventEmitter.Listeners.searchListener (/usr/local/lib/node_modules/fb-messenger-cli/scripts/listeners.js:111:21)
    at emitOne (events.js:96:13)
    at EventEmitter.emit (events.js:191:7)
    at ReadStream.InteractiveCli.handler (/usr/local/lib/node_modules/fb-messenger-cli/scripts/interactive.js:291:13)
    at emitOne (events.js:96:13)
    at ReadStream.emit (events.js:191:7)
    at readableAddChunk (_stream_readable.js:178:18)
    at ReadStream.Readable.push (_stream_readable.js:136:10)
    at TTY.onread (net.js:560:20)
```

**Comment by SamBergeron** [comment]
I just looked this up, it's not that lowercase search doesn't work. 
It's that if you try and do two searches in a row the second one crashes.
I'll try and get a fix for this tonight. Thanks.

**Comment by SamBergeron** [comment]
Closed with 118edb95149552aa6c8342ab0317ed526cda1d67
